superset_config.py

```diff
@@ -29,4 +29,15 @@
 AUTH_USER_REGISTRATION = True
 
 # The default user self registration role
-AUTH_USER_REGISTRATION_ROLE = "Gamma"
\ No newline at end of file
+AUTH_USER_REGISTRATION_ROLE = "Gamma"
+
+# Any other additional roles to be assigned to the user on top of the base role
+# Note: by design we cannot use AUTH_USER_REGISTRATION_ROLE to
+# specify more than one role
+AUTH_USER_ADDITIONAL_ROLES = ["sql_lab"]
+
+try:
+    # Overwrite repo settings with local settings
+    from local_config import *
+except:
```

Comment:
I think we should catch `ModuleNotFound` explicitly here. Reason being that if there's some logic in the `local_config.py` module that is erroneous, it will be masked by this `except` clause, resulting in unexpected behaviour.